packages/terra-table/tests/jest/Cell.test.jsx

```diff
@@ -8,6 +8,29 @@ import GridContext, { GridConstants } from '../../src/utils/GridContext';
 import Cell from '../../src/subcomponents/Cell';
 import ColumnContext from '../../src/utils/ColumnContext';
 
+const originalOffsetHeight = Object.getOwnPropertyDescriptor(
```

Comment:
This is required so that the elements will have an offsetWidth and offsetHeight in Jest.  Without this logic, the values will be 0 and the tests will fail.